operation/fhir-operation-bulkdata/src/main/java/com/ibm/fhir/operation/bulkdata/ExportOperation.java

```diff
@@ -64,10 +57,14 @@ protected Parameters doInvoke(FHIROperationContext operationContext, Class<? ext
         COMMON.checkAllowed(operationContext, false);
 
         // Pick off parameters
+        javax.ws.rs.core.UriInfo uriInfo = (javax.ws.rs.core.UriInfo) operationContext.getProperty(FHIROperationContext.PROPNAME_URI_INFO);
+        MultivaluedMap<String, String> queryParameters = uriInfo.getQueryParameters();
+
         MediaType outputFormat = export.checkAndConvertToMediaType(parameters);
 
         Instant since = export.checkAndExtractSince(parameters);
-        List<String> types = export.checkAndValidateTypes(parameters);
+
+        List<String> types = export.checkAndValidateTypes(parameters, queryParameters);
```

Comment:
I thought that our operations framework automatically converted the query parameters into a Parameters object.
Maybe it just doesn't merge the two if we receive both?